internal/locate/region_request.go

```diff
@@ -1752,6 +1752,9 @@ func (s *RegionRequestSender) onRegionError(
 				return true, nil
 			}
 		}
+		logutil.BgLogger().Error("Region is in flashback progress",
+			zap.Stringer("req", req),
```

Comment:
I'm inclined to contain these info inside the returned error, also consider the req can be nil.

Also, `req.String()` return weird things that no meaningful information is inside it. Maybe we should format some key fields of it.

Reply:
Updated.